.vortex/installer/tests/Fixtures/install/provision_database_lagoon/docs/onboarding.md

```
@@ -0,0 +1,18 @@
+@@ -80,6 +80,17 @@
+ 
+ ## 4. Setting up integrations
+ 
++- [ ] Configure Lagoon integration:
++  - [ ] Submit a request to AmazeeIO to create a project.
```

markdownlint-cli2 (0.17.2)

Unordered list style
Expected: plus; Actual: dash

(MD004, ul-style)